app/src/main/java/com/github/se/wanderpals/ui/screens/trip/Trip.kt

```diff
@@ -0,0 +1,66 @@
+package com.github.se.wanderpals.ui.screens.trip
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.BottomAppBarDefaults.containerColor
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.NavigationBar
+import androidx.compose.material3.NavigationBarDefaults
+import androidx.compose.material3.NavigationBarItem
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.contentColorFor
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.navigation.NavDestination.Companion.hierarchy
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.github.se.wanderpals.ui.navigation.NavigationActions
+import com.github.se.wanderpals.ui.navigation.Route
+import com.github.se.wanderpals.ui.navigation.TRIP_DESTINATIONS
+
+/** The Trip screen. */
+@Composable
+fun Trip(oldNavActions: NavigationActions) {
+  val navController = rememberNavController()
+  val navActions = NavigationActions(navController)
+
+  Scaffold(
+      modifier = Modifier.testTag("tripScreen"),
+      topBar = {},
+      bottomBar = { BottomBar(navActions) },
+      floatingActionButton = {}) { innerPadding ->
```

Comment:
floatingActionButton should be removed if not used later

Reply:
Yes, will do